fighters/gamewatch/src/status.rs

```diff
@@ -0,0 +1,53 @@
+use super::*;
+use globals::*;
+
+mod landing_attack_air;
+
+// #[smashline::fighter_init]
+// fn gamewatch_init(fighter: &mut L2CFighterCommon) {
+//     unsafe {
+//         // set the callbacks on fighter init
+//         if fighter.kind() == *FIGHTER_KIND_GANON {
+//             fighter.global_table[globals::USE_SPECIAL_N_CALLBACK].assign(&L2CValue::Ptr(should_use_special_n_callback as *const () as _));
+//             fighter.global_table[globals::STATUS_CHANGE_CALLBACK].assign(&L2CValue::Ptr(change_status_callback as *const () as _));   
+//         }
+//     }
+// }
+
+#[status_script(agent = "gamewatch", status = FIGHTER_STATUS_KIND_LANDING_ATTACK_AIR, condition = LUA_SCRIPT_STATUS_FUNC_STATUS_PRE)]
+pub unsafe fn landing_attack_air_pre(fighter: L2CFighterCommon) -> L2CValue {
```

Comment:
This should be moved into `landing_attack_air.rs` and replace the  one that's already there